src/styles/tooltip.less

```diff
@@ -2,37 +2,56 @@
   .sn-org-tooltip {
     visibility: hidden;
     opacity: 0;
-    font-family: 'QlikView Sans', sans-serif;
-    width: 240px;
-    max-height: 100px;
-    background-color: #404040;
-    color: #E6E6E6;
-    text-align: center;
-    font-size: 12px;
-    line-height: 15px;
-    padding: 5px 0;
-    border-radius: 6px;
-
-    overflow-wrap: break-word;
-    word-wrap: break-word;
-    word-break: break-word;
+    /* Unsupported in IE, so there we set width explicitly */
+    width: 230px;
+    width: -moz-max-content;
+    width: fit-content;
```

Comment:
Unless the width is set to something, it will not allow the tooltip to expand beyond the chart. This would be solved by moving the element to the body, but we'll leave that for later. In chrome/safari you can set fit-content and it behaves nicely, in firefox -moz-max-content does almost the same thing. In IE I used a fixed width instead, which isn't nice but fine.

Reply:
I was using firefox, looks good now :)